Rakefile

```diff
@@ -8,4 +8,12 @@ PuppetLint.configuration.send('disable_class_inherits_from_params_class')
 PuppetLint.configuration.send('disable_documentation')
 PuppetLint.configuration.send('disable_single_quote_string_with_variables')
 PuppetLint.configuration.send('disable_only_variable_string')
-PuppetLint.configuration.ignore_paths = ["spec/**/*.pp", "pkg/**/*.pp"]
+
+exclude_paths = [
+"pkg/**/*",
+"spec/**/*",
+]
+
+Rake::Task[:lint].clear
+PuppetLint.configuration.ignore_paths = exclude_paths
+
```

Comment:
Does this block need to go at the top of the file to make sure the other configurations aren't cleared?